infractions.yml

```diff
@@ -1,36 +1,48 @@
 ---
+# Positive associations with "white" (like whitelisted) and negative associations with "black"
+# (like blacklisted) recall a long legacy of violence and oppression against Black and
+# African-American populations. Prefer terms that do not ascribe positive or negative meaning to
+# colors.
 - terms:
   - whitelist
   - white-list
   alternatives:
   - allowlist
-
 - terms:
   - blacklist
   - black-list
   alternatives:
   - denylist
 
+# Grandfather clauses were included in discriminatory laws in the United States. The laws made it
+# more difficult for African Americans to register to vote. Prefer describing exemptions and
+# exceptions without recalling this history.
 - terms:
   - grandfather
   alternatives:
   - exempted
 
+# This term has been used as a slur to refer to individuals with hearing disabilities. Prefer more
+# descriptive alternatives for describing an object that is standing in for something else.
 - terms:
   - dummy
   alternatives:
   - placeholder
   - fake
   - stub
 
-# This term can be appropriate when used in the context of stress testing.
+# The term "redlining" is rooted in discriminatory practices in the United States of denying
+# services like mortgages or insurance based on geographical locations. Describe annotations on
+# designs using different terminology.
 - terms:
   - redline
   - red-line
   - redlining
   alternatives:
   - annotate
 
+# "Master–slave" is an offensive and exclusionary metaphor that cannot be detached from American
+# history. Describe a hierarchical relationship between nodes or objects with different terms.
```

Comment:
nit: Same comment as above - perhaps the second sentence can start with "Prefer describing a...". Thoughts?

Reply:
f17cab7